### ISSUE TITLE
Añadir sección e-commerces?

### ISSUE THREAD

**Issue by gabrielperales** [MurciaDev/murcia-tech-hub#6]
Técnicamente, PCcomponentes no es una empresa de producto… quizás deberíamos añadir la sección de e-commerces y añadir alguna como embargos a lo bestia, que también he visto que buscaban algún desarrollador últimamente … 🤔

**Comment by gabrielperales** [comment]
https://newgarden.es/en/

https://www.embargosalobestia.com

https://tahecosmetics.com/

**Comment by jgcarrillo** [comment]
Acabo de leerte! Justo he abierto una PR añadiendo a Eurobikes que también es e-commerce. En cualquier caso siempre se puede cambiar de sitio.

**Comment by cecicifu** [comment]
Yo creo que una cosa son empresas tech, y otras, empresas con ofertas tech, pero ya como lo queráis enfocar..

**Comment by gabrielperales** [comment]
@cecicifu sí, por eso, no sé si meterlas o no 🤷 . Por un lado, imagino que cuantas más empresas localizadas, mejor. Por otro lado no sé si formarían parte del _tech hub_. PCComponentes está claro que sí, porque han desarrollado su propio e-commerce y sí que tienen un departamento tecnológico grande.

**Comment by cecicifu** [comment]
@gabrielperales en el caso de Pccomponentes si, pero las que tienen un departamento muy pequeño de IT y encima no tienen su propio software no las incluiría, ya que probablemente sean muchísimas, o lo mismo no son tantas, como te digo, lo que veáis más apropiado, yo voy a intentar aportar todo lo que pueda en base a como lo vayáis definiendo. 😊
